notebooks/en/fine_tuning_llm_grpo_trl.ipynb

```diff
@@ -0,0 +1,3950 @@
+{
```

Comment:
<p>Just to be clear, even if in the R1 paper, the completion length increases, there is no incentive, like explicit reward for this.</p><p>I think you should remove this length reward (the completion length is logged anyway if you want to monitor the completion length)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>